jenkins/k8s/jenkins.yaml

```diff
@@ -30,6 +30,14 @@ spec:
         ports:
         - containerPort: 8080
         - containerPort: 50000
+        readinessProbe:
+          httpGet:
+            path: /login
+            port: 8080
+          periodSeconds: 1
+          timeoutSeconds: 1
```

Comment:
Can we relax these a bit. 

I would say a period of 5-10 seconds would be more reasonable. Timeout at around 2-5 seconds. Failure threshold at 5 and success threshold at 2.

WDYT?